hw/ip/csrng/data/csrng.hjson

```diff
@@ -212,6 +212,46 @@
         }
       ]
     },
+    { name: "HALT_MAIN_SM",
+      desc: "Halt the CSRNG main state machine register",
+      swaccess: "wo",
+      hwaccess: "hro",
+      regwen: "REGWEN",
+      fields: [
+        {
+            bits: "0",
+            name: "HALT_MAIN_SM",
+            desc: '''
+                  Setting this bit halt the main state machine that processes
```

Comment:
nit: Setting this bit halt**s**...

Reply:
good catch...fix this wording as well as some related text.